lib/src/main/java/net/cachapa/expandablelayout/ExpandableLinearLayout.java

```diff
@@ -222,6 +234,24 @@ private void animateHeight(final View view, final int targetHeight) {
         lp.weight = 0;
         int height = view.getHeight();
 
+        if (!animate) {
```

Comment:
I'm not a fan of return statements in the middle of methods, as it makes it hard to follow the code path.
The non-animated option should be separated into its own function.

Since I was meaning to refactor this code anyhow, I'm going to accept this pull request and refactor on top of it.